tests/compilation_tests/robot_test/robot_test.py

```diff
@@ -41,15 +43,26 @@ def test_travel(self):
         new_x = round(-1.3662, 3)
         new_y = round(-0.366198, 3)
         new_theta = round(3 * math.pi / 2, 3)
-        self.assertEqual([[float(new_x)], [float(new_y)], [float(new_theta)]], self.robot_one.robot_state.state.tolist())
-
-    def test_move_forward_default(self):
-        # Values calculated by hand based on kinematic equations
-        self.robot_two.move_forward(self.distance) #self.distance = 10
+        self.assertEqual([[float(new_x)], [float(new_y)], [float(
+            new_theta)]], self.robot_one.robot_state.state.tolist())
+        # was move_forward_default
+        self.robot_two.travel(self.distance, 0)
         new_x = self.robot_two.robot_state.state[0]
         new_y = self.robot_two.robot_state.state[1]
         new_theta = float(self.robot_two.robot_state.state[2])
-        self.assertEqual([[float(new_x)], [float(new_y)], [float(new_theta)]], self.robot_two.robot_state.state.tolist())
+        self.assertEqual([[float(new_x)], [float(new_y)], [float(
```

Comment:
is there a reason why these are all within the same test case instead of breaking it up into separate ones?